source/api.rst

```diff
@@ -0,0 +1,626 @@
+
+.. code-block:: bash
+
+   None
+
+**Response:**
+
+.. container:: toggle
+
+   .. container:: header
+
+      JSON:
+
+   .. include:: ./assets/code/api-list-cluster-response-json.txt
+
+Get status of PXC cluster
+-------------------------
+
+**Description:**
+
+.. code-block:: bash
+
+   Gets all information about specified pxc cluster
+
+**Kubectl Command:**
+
+.. code-block:: bash
+
+   kubectl --v=9 get pxc/cluster1 -o json
+
+**URL:**
+
+.. code-block:: bash
+
+   https://$API_SERVER/apis/pxc.percona.com/v1/namespaces/default/perconaxtradbclusters/cluster1
+
+**Authentication:**
+
+.. code-block:: bash
+
+   Authorization: Bearer $KUBE_TOKEN
+
+**cURL Request:**
+
+.. code-block:: bash
+
+   curl -k -v -XGET "https://$API_SERVER/apis/pxc.percona.com/v1/namespaces/default/perconaxtradbclusters/cluster1" \
+               -H "Accept: application/json" \
+               -H "Authorization: Bearer $KUBE_TOKEN"
+
+**Request Body:**
+
+.. code-block:: bash
+
+   None
+
+**Response:**
+
+.. container:: toggle
+
+   .. container:: header
+
+      JSON:
+
+   .. include:: ./assets/code/api-get-status-of-cluster-response-json.txt
+
+Scale up/down PXC cluster
+-------------------------
+
+**Description:**
+
+.. code-block:: bash
+
+   Increase or decrease the size of the PXC cluster nodes to fit the current high availability needs
+
+**Kubectl Command:**
+
+.. code-block:: bash
+
+   kubectl --v=9 patch pxc cluster1 --type=merge --patch '{
+   "spec": {"pxc":{ "size": "5" }  
+   }}'
+
+**URL:**
+
+.. code-block:: bash
+
+   https://$API_SERVER/apis/pxc.percona.com/v1/namespaces/default/perconaxtradbclusters/cluster1
+
+**Authentication:**
+
+.. code-block:: bash
+
+   Authorization: Bearer $KUBE_TOKEN
+
+**cURL Request:**
+
+.. code-block:: bash
+
+   curl -k -v -XPATCH 'https://$API_SERVER/apis/pxc.percona.com/v1/namespaces/default/perconaxtradbclusters/cluster1' \
+               -H "Authorization: Bearer $KUBE_TOKEN" \
+               -H "Content-Type: application/merge-patch+json" 
+               -H "Accept: application/json" \
+               -d '{  
+                     "spec": {"pxc":{ "size": "5" }  
+                     }}'
+
+**Request Body:**
+
+.. container:: toggle
+
+   .. container:: header
+
+      JSON:
+
+   .. include:: ./assets/code/api-scale-cluster-request-json.txt
+
+**Input:**
+
+   **spec**:
+
+   pxc
+
+   1. size (Int or String, Defaults: 3): ``Specifiy the sie of the pxc cluster to scale up or down to``
+
+**Response:**
+
+.. container:: toggle
+
+   .. container:: header
+
+      JSON:
+
+   .. include:: ./assets/code/api-scale-cluster-response-json.txt
+
+Update PXC cluster image
+------------------------
+
+**Description:**
+
+.. code-block:: bash
+
+   Change the image of pxc containers inside the cluster
+
+**Kubectl Command:**
+
+.. code-block:: bash
+
+   kubectl --v=9 patch pxc cluster1 --type=merge --patch '{  
+   "spec": {"pxc":{ "image": "percona/percona-xtradb-cluster-operator:1.5.0-pxc5.7" }  
```

Comment:
as you know we do not have  such image starting from 1.5.0.  Please use `percona/percona-xtradb-cluster:5.7.30-31.43`